activiti-services/activiti-services-events/src/main/java/org/activiti/services/events/MessageProducerCommandContextCloseListener.java

```diff
@@ -0,0 +1,48 @@
+package org.activiti.services.events;
+
+import java.util.List;
+
+import org.activiti.engine.impl.context.Context;
+import org.activiti.engine.impl.interceptor.CommandContext;
+import org.activiti.engine.impl.interceptor.CommandContextCloseListener;
+import org.activiti.services.api.events.ProcessEngineEvent;
+import org.springframework.messaging.support.MessageBuilder;
+
+public class MessageProducerCommandContextCloseListener implements CommandContextCloseListener {
```

Comment:
Thank you for your advice. I fixed.
https://github.com/Activiti/Activiti/pull/1424/commits/5d74bf002f8b8537147b9b0096feb4bc415b659f